os/fs/driver/mtd/smart.c

```diff
@@ -4895,6 +4919,10 @@ static int smart_ioctl(FAR struct inode *inode, int cmd, unsigned long arg)
 		case SMART_DEBUG_CMD_DUMP_PSECTOR:
 			ret = smart_dumpsector(dev, (uint16_t)debug_data->debugdata, debug_data->debugcmd);
 			goto ok_out;
+
+		case SMART_DEBUG_CMD_DUMP_ALL:
```

Comment:
this command doesn't bring all sectors just one time...
SMART_DEBUG_COPY_PSECTOR or something else to explain that is copy specific sector data from flash..

Reply:
Lets rename this to `SMART_DEBUG_CMD_RETRIEVE_SECTOR` as it retrieves the data of one sector at a time